bookbag/workshop/content/perform-trace-analysis.adoc

```diff
@@ -138,55 +172,47 @@ image::images/catalog-controller-updated.png[Catalog Controller Updated- Java co
 Do not forget to import the missing packages.
 ====
 
-Now let's test the new version of the source code.
-To do it we will use a new Project framework called Istio Workspace.
-
-'''
-
-=== What is Istio Workspace?
-
-[sidebar]
---
-image::images/istio-workspaces-logo.png[Istio Workspace, 400]
-
-https://github.com/Maistra/istio-workspace[Istio Workspace^] : 
-Safely develop and test on any kubernetes cluster without distracting others.
+With those changes in place we can now test it. But does it mean we have to go through the whole **build -> deploy -> test** cycle? That can take a couple of minutes.
+Luckily, with Istio Workspace we can now start service with our changes locally, but in a way like it would be running as part of the application.
 
-* Do you have confidence that your Test and Stage environments reflect reality? 
-* Are they easy to keep up-to-date with production? 
-* Can you reproduce production failure in these environments or locally? 
-* What about that error you keep seeing on production that’s never occurring on Test or Stage?
+[source,shell,subs="{markup-in-source}"]
 
-It’s been always hard to test new functionality before it reaches production. Even more so, with the shift from a **monolith** to **microservices** and increasing scale. 
-Can you still spin up the entire solution on your laptop to debug and test a suspicious piece of code? Testing on production is no longer a meme. It’s reality and a necessity.
+.>_ workshop-tools terminal
 
-This project works with Istio and Kubernetes or Openshift to give you confidence that your changes won’t blow up production cluster and your users won’t even notice a glitch.
+$ ike develop --deployment catalog-v1 --run "mvn spring-boot:run" --port 9000:8080 --route header:ike-session-id=lab3
```

Comment:
Move this to a DevFile command?

Reply:
I will add that, sure

Reply:
But I guess @mcouliba wants to clean it up first anyway.